crates/solidity/testing/sanctuary/src/tests.rs

```diff
@@ -73,6 +73,8 @@ pub fn run_test(file: &SourceFile, events: &Events) -> Result<()> {
     let source = file
         .path
         .read_to_string()?
+        // Some files are null terminated, remove the null character:
```

Comment:
How are handling it right now? Is the parser not resilient enough to handle it?

Reply:
No, because it is not part of our grammar, and is not supported by `solc`.
I suspect it is an issue with how `sanctuary` stores files. I filed an issue here and added it as a comment:

https://github.com/tintinweb/smart-contract-sanctuary/issues/30